CHANGELOG.md

```diff
@@ -102,23 +102,23 @@ All notable changes to this project will be documented in this file.
 -   \[INTERNALS\] Remove unused functions `Errors.unset_target`, and `Errors.reset_target`.
 
 ## \[1.12.1\] - 2024-09-25
```

Comment:
Date to update before merge

Reply:
Please go ahead and set it to today

Reply:
Done [30f5cc6](https://github.com/pyccel/pyccel/pull/1997/commits/30f5cc62a7beea953e0769bfa37a2e6f8d9a4953)